flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Refuel.js

```diff
@@ -68,7 +67,7 @@ class A32NX_Refuel {
         const LOutTarget = LOutTargetSimVar;
         const RInnTarget = RInnTargetSimVar;
         const ROutTarget = ROutTargetSimVar;
-        if (refuelRate == 2) {
+        if (refuelRate == '2') {
```

Comment:
**suggestion:** Since we are gonna rewrite this, and don't wanna refactor too much, can we add a comment saying what that magic value is ?